api/workAllocation/index.ts

```diff
@@ -63,8 +63,7 @@ export async function searchTask(req: EnhancedRequest, res: Response, next: Next
     res.status(status);
     // Assign actions to the tasks on the data from the API.
     if (data) {
-      const caseworkers: Caseworker[] = await retrieveAllCaseWorkers(req, res);
-      assignActionsToTasks(data.tasks, req.body.view, caseworkers);
+      assignActionsToTasks(data.tasks, req.body.view, req.session.caseworkers);
```

Comment:
I already requested this be changed on https://github.com/hmcts/rpx-xui-webapp/pull/953. Why was this merged? 🤯

Reply:
This has been merged after discussing with @ronaldmansveld  that this is not an issue. I can still explain you if you have the time. Thanks.

Reply:
As per call This is agreed to be revisited. Can you approve it now? Thanks

Reply:
https://tools.hmcts.net/jira/browse/EUI-3611